receiver/kubeletstatsreceiver/metadata.yaml

```diff
@@ -187,6 +187,48 @@ metrics:
       monotonic: true
       aggregation_temporality: cumulative
     attributes: []
+  k8s.node.runtime.imagefs.available:
+    enabled: true
+    description: Node runtime imagefs available
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+  k8s.node.runtime.imagefs.capacity:
+    enabled: true
+    description: Node runtime imagefs capacity
+    unit: By
+    gauge:
+      value_type: int
+    attributes: []
+  k8s.node.runtime.imagefs.used:
+    enabled: true
+    description: Node runtime imagefs used
```

Comment:
Can you include in the description that this value may not equal `CapacityBytes - AvailableBytes.`

Reply:
I am very sorry for my foolish mistake that requires me to resubmit the pull request.Thank you for reviewing the code. https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26259